web/frontend/components/PublishButton.vue

```diff
@@ -1,45 +1,110 @@
-    <button type="button" 
-            class="action publish one-line" 
-            :disabled="disabled"
-            :title="disabled ? 'Some resources are incomplete. Finalize Entries below.' : 'Publish'"
-            @click="attemptPublish">
-            Publish
+    <button
+      type="button"
+      class="action publish one-line"
+      :disabled="disabled"
+      :title="
+        disabled
+          ? 'Some resources are incomplete. Finalize entries below.'
+          : 'Publish'
+      "
+      @click="attemptPublish"
+    >
+      Publish
     </button>
+
     <Modal v-if="showModal" @close="showModal = false">
-      <template slot="title">Confirm Publish</template>
+      <template slot="title">
+        <div class="publish-modal">
+          {{ publishSuccess ? "Your book is published!" : "Publishing your book" }}
+        </div>
+      </template>
       <template slot="body">
-        <p>Are you ready to publish your book?</p>
-        
-        <div v-if=publishCheck.isVerifiedProfessor id="prof-prompt">
-          You're almost ready to publish your book! The following elements are
-          optional, but can make your book more discoverable by students and
-          colleagues:
-          <ul class="fa-ul">
-            <li v-if="this.publishCheck.coverImageFlag">
-              <span v-if="this.publishCheck.coverImageExists" class="fa-li"><font-awesome-icon icon="fa-solid fa-check fa-fw" /></span>
-              <span v-else class="fa-li"><font-awesome-icon icon="fa-solid fa-xmark fa-fw" /></span>
-              Cover Photo
-            </li>
-            <li>
-              <span v-if="this.publishCheck.descriptionExists" class="fa-li"><font-awesome-icon icon="fa-solid fa-check fa-fw" /></span>
-              <span v-else class="fa-li"><font-awesome-icon icon="fa-solid fa-xmark fa-fw" /></span>
-              Description 
-            </li>
-          </ul>
-          We also encourage you to share your work with your colleagues or on
-          social media! Your book is an important contribution to open education
-          for law and we hope you'll help others find it and use it in their own
-          classrooms. Because you are a verified professor, your book will
-          automatically be surfaced on H2O's search page and will appear in web
-          searches.
+        <aside class="publish-modal">
+        <div v-if="!publishSuccess">
+          <div v-if="publishCheck.isVerifiedProfessor" id="prof-prompt">
+            <p>
+              You're almost ready to publish your book! The following elements
+              are optional, but can make your book more discoverable by students
+              and colleagues:
+            </p>
+            <ul class="fa-ul">
+              <li v-if="this.publishCheck.coverImageFlag">
+                <span v-if="this.publishCheck.coverImageExists" class="fa-li"
+                  ><font-awesome-icon icon="fa-solid fa-check fa-fw"
+                /></span>
```

Comment:
Clearly this is some auto-formatter deciding where to put line breaks in this HTML, but I'm really curious what it's thinking. Any guesses why it wants
```
<span v-if="this.publishCheck.coverImageExists" class="fa-li"
  ><font-awesome-icon icon="fa-solid fa-check fa-fw"
/></span>
```
instead of
```
<span v-if="this.publishCheck.coverImageExists" class="fa-li">
  <font-awesome-icon icon="fa-solid fa-check fa-fw"/>
</span>
```

Reply:
It's a Vue mode for VS Code but no idea what its heuristic was here (I agree it's weird). I'll change it.